hedera-mirror-rest/controllers/networkController.js

```diff
@@ -242,6 +251,26 @@ class NetworkController extends BaseController {
     res.locals[constants.responseDataLabel] = new NetworkSupplyViewModel(rows[0], NetworkController.totalSupply);
     logger.debug(`getSupply returning ${rows.length} entries`);
   };
+
+  /**
+   * Handler function for /network/fees API.
+   * @param {Request} req HTTP request object
+   * @param {Response} res HTTP response object
+   * @return {Promise<void>}
+   */
+  getFees = async (req, res) => {
+    const filters = utils.buildAndValidateFilters(req.query);
+    const {filterQuery, order} = this.extractFileDataQuery(filters);
+
+    const exchangeRate = await FileDataService.getExchangeRate(filterQuery);
```

Comment:
Combine to make this call in parallel
```suggestion
    const [exchangeRate, feeSchedule] = await Promise.all([
      FileDataService.getExchangeRate(filterQuery),
      FileDataService.getExchangeRate(filterQuery)
    ]);
```